### PR TITLE
CE-3054 Add classification of the first-level templates as well

### PR DESCRIPTION
**Pull request by adamkarminski**
/cc: @SebastianMarzjan

### CODE REVIEW

File: extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php [hunk 1]
```diff
@@ -82,20 +83,23 @@ public function execute() {
...
-							$wgCityId,
-							$templateId,
-							TemplateClassificationService::TEMPLATE_NOT_ART,
-							self::NONARTICLE_MAINTENANCE_PROVIDER,
-							$origin
-						);
-					} catch ( \Swagger\Client\ApiException $e ) {
-						$this->out( 'Classification failed!' );
-					}
+				$type = TemplateClassificationService::TEMPLATE_NOT_ART;
```
Comment (sebastianmarzjan) [inline extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php:89]: Why so?
Some templates are indeed works of art! :smiling_imp:

File: extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php [hunk 2]
```diff
@@ -82,20 +83,23 @@ public function execute() {
...
+				try {
+					$tcs->classifyTemplate(
+						$wgCityId,
+						$templateId,
+						$type,
+						self::NONARTICLE_MAINTENANCE_PROVIDER,
+						$origin
+					);
+				} catch ( \Swagger\Client\ApiException $e ) {
+					$this->out( 'Classification failed!' );
```
Comment (sebastianmarzjan) [inline extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php:102]: Will you see it or maybe it's worth to log it to Kibana?

File: extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php (hunk 1 above)
Comment (adamkarminski) [inline extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php:89]: More of them are a work of the devil! :imp:

File: extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php (hunk 2 above)
Comment (lgarczewski) [inline extensions/wikia/TemplateClassification/maintenance/classifyNonArticleTemplates.class.php:102]: Meh, for a one time run like this I'd say just showing output is fine.

**Comment by lgarczewski** [comment]
LGTM but:

Please test what the service returns with these data in, for both types! We don't want to screw up Mercury or Human TC display once this gets into the DB.

**Pull request merged**